topics/security/index.md

```diff
@@ -6,4 +6,4 @@ short_description: Cybersecurity is the protection of computer hardware, softwar
 topic: security
 wikipedia_url: https://en.wikipedia.org/wiki/Computer_security
 ---
-Cybersecurity is the protection of computer hardware, software, and data from theft, damage, and malicious intent.
+Cybersecurity is the protection of computers from theft and damage to their hardware, software, or information. It also involves protection from disruption or misdirection of the services they provide. Cybersecurity includes controlling physical access to hardware, as well as protection from attacks that come via network access, data injection, and code injection.
```

Comment:
This is ✨ , maybe drop the `,` here: `Cybersecurity includes controlling physical access to hardware, as well as`